src/FischBot/Services/HalfMastService/HalfMastService.cs

```diff
@@ -39,24 +40,22 @@ public HalfMastService(IConfiguration configuration)
         public async Task<HalfMastNotice> GetLatestHalfMastNotice(string state)
         {
             var searchState = string.IsNullOrEmpty(state) ? "Entire United States" : state;
-            var response = await _twitterClient.SearchV2.SearchTweetsAsync($"(from: {_halfMastAlertsTwitterAccountName}) {searchState}");
+            var response = await _twitterClient.SearchV2.SearchTweetsAsync($"(from: {HalfMastAlertsTwitterAccountName}) {searchState}");
             var latestHalfMastTweet = response.Tweets.FirstOrDefault();
 
-            if (latestHalfMastTweet != null) 
+            if (latestHalfMastTweet != null)
             {
-                var halfMastNotice = new HalfMastNotice() 
+                var halfMastNotice = new HalfMastNotice()
                 {
-                    State = Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(searchState),
+                    State = WebUtility.HtmlDecode(Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(searchState)),
                     Description = latestHalfMastTweet.Text,
-                    SourceUrl = $"https://twitter.com/{_halfMastAlertsTwitterAccountName}/status/{latestHalfMastTweet.Id}"
+                    SourceUrl = $"https://twitter.com/{HalfMastAlertsTwitterAccountName}/status/{latestHalfMastTweet.Id}"
                 };
 
                 return halfMastNotice;
             }
-            else 
-            {
-                return null;
-            }
+
+            return null;
```

Comment:
please keep this in the `else` block to match the style of the rest of the project